Documentation/Signoffs/jiegwuatu42-web_wireless-signoff.md

```diff
@@ -7,22 +7,36 @@ The Wireless acquisition of data helps effortlessly boost the quality of life. T
 
 The subsystem constraints are placed to ensure that it can fulfill its job in a satisfactory manner. Constraints on the subsystem are as follows:
```

Comment:
1. The quantity of data needs to be specified. 2. The algorithm cannot be both undisclosed and be specified as AES Symmetric Encryption. Where the data is encrypted and decrypted is not specified. (It would help to specify the purpose of the encryption.) 3. Not clear...do you mean accessible via the Internet? 4. Duplicate of #1? 5. Not sure what this means...would one ever want the data to be illegible?

Reply:
Constraints format - list the constraint first then provide the justification for the constraint.

1. It is not clear how 4G LTE, Wi-Fi, and Internet are related. Does the server need to deal with Wi-Fi and 4G LTE or does it just connect to the Internet?

3. Does this mean the microcontroller must implement encryption? If so, does that appear in the microcontroller subsystem constraints? Also how will the user implement the decryption when using a web browser?

4. How can this constraint be evaluated? Exactly what is meant by "process"?

5. OK but it seems that both sentences say the same thing. If they mean different things it needs to be clarified.

6. The intent is clear but the specification is vague. The phrase "easy-to-understand" is subjective.

Reply:
Note: the weird reference to create microcontroller in a previous comment is from a github idiosyncracy. That was supposed to refer to constraint number 1.

For every constraint be sure to list the constraint then list the justification for it.

Number 1 and 5  seem to be saying similar things. It's probably best to separate them into two, one being communication with the measurement system and the other being communication to the user (via a web page or however that will occur).

Number 3. I don't see a reference number 9. (Why the change to TLS? I'm not complaining about it...just curious.) Clarify the two times, in particular what is a "communication"?

Number 4. Is "read, decrypt, and format" a constraint? Encryption is already covered. Reading the data is implied by other constraints (it can't be displayed to the user if it can't be read). Formatting the data would be part of displaying the data to the user, unless you mean something else by format.

Number 6. I'm not sure that "user friendly" is much of an improvement...but "graphical interface" is at least a bit more specific. Not sure what to recommend here but try to be a bit more specific. Is the user interface intended to be a web page? If so, specify that. (Hint: there's a standard that can be referenced....)